### ISSUE TITLE
Bundler could not find compatible versions for gem "activerecord":  error 

### ISSUE THREAD

**Issue by gorakrao** [Bodacious/blogit#91]
Bundler could not find compatible versions for gem "activerecord":
  In snapshot (Gemfile.lock):
    activerecord (= 5.0.0)

  In Gemfile:
    acts-as-taggable-on (~> 3.5.0) was resolved to 3.5.0, which depends on
      activerecord (< 5, >= 3.2)

```
rails (~> 5.0.0) was resolved to 5.0.0, which depends on
  activerecord (= 5.0.0)
```

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

**Comment by rhizome** [comment]
No, "activerecord (< 5, >= 3.2)" means it wants a Rails version less than 5 but greater than 3.2. Check to see if you have Rails 5 in your Gemfile.

**Comment by ruipin** [comment]
@rhizome From what I understand, it's because blogit depends on acts-as-taggable-on ~> 3.5.0, which is incompatible with Rails 5.
As such, for blogit to be compatible with Rails 5, the project needs to update to acts-as-taggable-on 4.0
This is the same issue as #92
